packages/eslint-config-base/test/samples/es2015-esm/contrived.bad.js

```diff
@@ -1,11 +1,25 @@
 // Made up sample.. not genuine
-
 // Test unnecessary `use strict`
-
```

Comment:
Add this new line back to remove the `lines-around-directive` error from the snapshot.

Reply:
Fixed!